stdlib/Test/src/Test.jl

```diff
@@ -718,6 +718,110 @@ function get_test_result(ex, source)
             $(QuoteNode(source)),
             $negate,
         ))
+    elseif isa(ex, Expr) && (ex.head === :|| || ex.head === :&&) && length(ex.args) == 2
```

Comment:
I think these should use `Base.isexpr`.

Reply:
I copied the other branches in this function. Should they all be changed?